proxywasm/internal/abi_callback_alloc.go

```diff
@@ -14,9 +14,15 @@
 
 package internal
 
-//nolint
+import "time"
+
+// nolint
+//
 //export proxy_on_memory_allocate
 func proxyOnMemoryAllocate(size uint) *byte {
+	if recordTiming {
+		logTiming("proxyOnMemoryAllocate", time.Now())
```

Comment:
need to add `defer` to make `time.Since` works as expected in logTiming?

```suggestion
		defer logTiming("proxyOnMemoryAllocate", time.Now())
```